.github/workflows/code_coverage.yml

```diff
@@ -4,31 +4,34 @@ name: Code Coverage
 
 # Controls when the action will run. Triggers the workflow on pull request
 # events or push events in the develop branch.
+#on:
+#  pull_request_target:
+#    types: [opened, synchronize, reopened]
```

Comment:
I'm seeing a bunch of commented out code in the latest commit--did you mean for that to be the case?

Reply:
Yeah, those were introduced in efforts to fix the issues with fork pushes earlier, reverting the introduction to `pull_request_target` here as they need to be moved to a separate upload comment workflow while still keeping the requirement to wait for unit tests commented out, just to help with verifying the functionality faster. Will undo them too when everything's finally verified.

Reply:
Reverted them as the PR wouldn't be able to verify the comment upload workflows.